tests/test_model_api.py

```diff
@@ -92,42 +93,36 @@ def test_set_equation(self, model):
 
         # Get model, assert that V is a state variable
         v = model.get_symbol_by_ontology_term(OXMETA, 'membrane_voltage')
-        v_meta = model.get_meta_dummy(v)
-        assert v_meta.type == 'state'
+        assert v.type == 'state'
 
         # Now clamp it to -80mV
-        v_unit = v_meta.units
-        rhs = model.add_number(number=sp.Number(-80), units=str(v_unit))
+        rhs = model.add_number(sp.Number(-80), str(v.units))
         model.set_equation(v, rhs)
 
         # Check that V is no longer a state
         v = model.get_symbol_by_ontology_term(OXMETA, 'membrane_voltage')
-        v_meta = model.get_meta_dummy(v)
-        assert v_meta.type != 'state'
+        assert v.type != 'state'
 
         # TODO: Get dvdt_unit in a more sensible way
         # See: https://github.com/ModellingWebLab/cellmlmanip/issues/133
 
         # Now make V a state again
         t = model.get_symbol_by_ontology_term(OXMETA, 'time')
-        t_meta = model.get_meta_dummy(t)
-        t_unit = t_meta.units
         lhs = sp.Derivative(v, t)
-        dvdt_unit = 'unlikely_unit_name'
-        model.add_unit(dvdt_unit, [
-            {'units': str(v_unit)},
-            {'units': str(t_unit), 'exponent': -1},
+        dvdt_units = 'unlikely_unit_name'
```

Comment:
Can't we do `dvdt_units = str(v.units / t.units)` by analogy with the connection handling in `Model`?

Reply:
Didn't see that bit!
In fact, it turns out the test fails if I do that because that unit name is already taken.

Reply:
Ah! I already made an issue for this #133 : The problem is there's no way (or at least I didn't see a way at the time) to check if our model knows about a particular unit, and then to get the string representation for it. This is needed because we specify units through their string names exclusively (makes sense for CellML parsing), so this issue would go away if we change that #132

Reply:
Let's leave this as-is until doing #132 then.